js/bookClass.js

```diff
@@ -41,7 +40,7 @@ class Book {
       if (e.target.classList.contains('remove')) {
         const list = e.target.parentElement;
         this.ul.removeChild(list);
-        const title = list.childNodes[5].value;
+        const title = list.childNodes[3].value;
         this.bookData.filter((book) => book.title !== title);
       }
     });
```

Comment:
- [ ] OPTIONAL: This `removeBook` method does not update the local storage when a book is removed from the list. Then when the page is refreshed it gets back to its previous state. I just made this comment optional just because it wasn't mentioned by the previous reviewer. It is important and I strongly advice you to fix this issue. :pray:

Reply:
@henatan99
Ohh yes i didn't notice that
I will fix them now 
Thank you so much